Dart/src/com/jetbrains/lang/dart/ide/runner/server/webdev/DartWebdevConfigurationType.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2000-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+package com.jetbrains.lang.dart.ide.runner.server.webdev;
+
+import com.intellij.execution.configurations.ConfigurationFactory;
+import com.intellij.execution.configurations.ConfigurationTypeBase;
+import com.intellij.execution.configurations.ConfigurationTypeUtil;
+import com.intellij.execution.configurations.RunConfiguration;
+import com.intellij.ide.highlighter.HtmlFileType;
+import com.intellij.openapi.project.DumbAware;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.NotNullLazyValue;
+import com.intellij.psi.search.FileTypeIndex;
+import com.intellij.psi.search.GlobalSearchScope;
+import com.jetbrains.lang.dart.DartBundle;
+import icons.DartIcons;
+import org.jetbrains.annotations.NotNull;
+
+public class DartWebdevConfigurationType extends ConfigurationTypeBase implements DumbAware {
+  @NotNull
+  public static DartWebdevConfigurationType getInstance() {
+    return ConfigurationTypeUtil.findConfigurationType(DartWebdevConfigurationType.class);
+  }
+
+  public DartWebdevConfigurationType() {
+    super("DartRemoteDebugConfigurationType",
+          DartBundle.message("webdev.debug.configuration.name"),
+          DartBundle.message("webdev.debug.configuration.description"),
+          NotNullLazyValue.createValue(() -> DartIcons.Dart_16));
+    addFactory(new ConfigurationFactory(this) {
+      @NotNull
+      @Override
+      public String getName() {
+        return "Dart Web"; // compatibility
```

Comment:
That's the first version of the new run configuration, there's nothing we need to be compatible with :)

Reply:
Removed